src/pages/auth/verify/index.tsx

```
@@ -0,0 +1,137 @@
+                  />
+                ) : (
+                  <div className="flex flex-col items-center gap-1 text-slate-400">
+                    <IconPhoto className="w-10 h-10" aria-hidden />
+                    <span className="text-xs">클릭 또는 드래그</span>
+                  </div>
+                )}
+
+                {slot.file && (
+                  <button
+                    type="button"
+                    onClick={(e) => { e.stopPropagation(); clearSlot(slot.id) }}
```

ESLint

[error] Replace `·e.stopPropagation();·clearSlot(slot.id)` with `⏎······················e.stopPropagation()⏎······················clearSlot(slot.id)⏎···················`

(prettier/prettier)